lib/IPv4.js

```diff
@@ -426,6 +426,25 @@ class IPv4 {
     return result;
   }
 
+  /**
+   * Get all ip addresses in ip network
+   * @param {string} network
+   * @returns {string[]}
+   */
+  getAllNetworkAddresses(network) {
+    if (!this.isNetworkValid(network)) {
+      throw new Error('Invalid IP network!');
```

Comment:
```suggestion
      throw new Error('Invalid ip network!');
```
fix error message style